sap/cli/gcts.py

```diff
@@ -940,36 +945,32 @@ def update_filesystem(connection, args):
         return 1
 
     repo = get_repository(connection, args.package)
-    checkout_progress = ConsoleSugarOperationProgress(console)
-    noimports_progress = ConsoleSugarOperationProgress(console)
+    shared_progress = ConsoleSugarOperationProgress(console)
     pull_response = None
-    errored = True
     try:
-        with abap_modifications_disabled(repo, progress=noimports_progress):
-            with temporary_switched_branch(repo, args.branch, progress=checkout_progress):
-                console.printout(f'Updating the currently active branch {args.branch} ...')
-                pull_response = repo.pull()
-                from_commit = pull_response.get('fromCommit') or '()'
-                to_commit = pull_response.get('toCommit') or '()'
-                console.printout(f'The branch "{args.branch}" has been updated: {from_commit} -> {to_commit}')
+        with (
+            abap_modifications_disabled(repo, progress=shared_progress),
+            temporary_switched_branch(repo, args.branch, progress=shared_progress),
+        ):
+            console.printout(f'Updating the currently active branch {args.branch} ...')
+            pull_response = repo.pull()
+            from_commit = pull_response.get('fromCommit') or '()'
+            to_commit = pull_response.get('toCommit') or '()'
+            console.printout(f'The branch "{args.branch}" has been updated: {from_commit} -> {to_commit}')
```

Comment:
I don't think you had to change these lines. Please, do no do unnecessary cosmetic changes.